scripts/upgrade-aws.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
This seems to be all that's needed? Or any missed steps here?

Reply:
I think that's correct. (Or at least that's all I had to do to update the version of the docker plugin a while back.)